code/iaas/config-item/server/src/main/java/io/cattle/platform/configitem/context/data/metadata/common/NetworkMetaData.java

```diff
@@ -1,22 +1,31 @@
 package io.cattle.platform.configitem.context.data.metadata.common;
 
+import java.util.List;
 import java.util.Map;
 
+import io.cattle.platform.core.addon.NetworkPolicyRule;
+import io.cattle.platform.core.addon.NetworkPolicyRule.NetworkPolicyRuleAction;
+
 public class NetworkMetaData {
     String name;
     String uuid;
     boolean is_default;
     boolean host_ports;
     protected Map<String, Object> metadata;
+    NetworkPolicyRuleAction default_policy_action;
```

Comment:
Just change to string, and List<Object>.  Using actual types just add unneeded overhead in marshaller/unmarshaller I believe.